README.md

```diff
@@ -1,86 +1,130 @@
+* Package Management: [Node Package Manager](https://www.npmjs.com) (npm)
 
-***
+* State Management: [React Redux](https://react-redux.js.org) for easier context passing of state
 
-## Example Client Features
-### Profile
-* Create a new profile
-* Edit your profile
-* See other users' profiles
-  - Publicly follow/unfollow other users
-* Display list of followers/following
+* Network Request: [Web3.js](https://web3js.readthedocs.io/en/v1.3.4/) for Ethereum blockchain interaction
 
-### Posts
-* Create a public post
-* View a public post in the feed
-* Comment on a public post
-  - Note: only to one degree of depth
-* React to posts
-* Filter and navigate through the feed
-  - My Posts
-  - My Feed
-  - A specific user's posts
-  - Main Feed
+* Frontend Framework:
+  - [ReactJS](https://reactjs.org) with [Typescript](https://www.typescriptlang.org)
+  - [SCSS](https://sass-lang.com) with [BEM](https://en.bem.info/methodology/css/) methodology
+  - Responsive with Desktop first priority
+  - Utilize media queries, SCSS mixing
+  - [Ant Design](https://ant.design) Library of pre-built React components
 
-#### Note
-The features of the Example Client are subject to change with future versions and as decisions are made on the DSNP SDK.
+* Testing:
+  - Tests written in [Jest](https://jestjs.io) and [Enzyme](https://enzymejs.github.io/enzyme/)
+  - CI/CD done with [Github Actions](https://github.com/features/actions)
 
-### Docker
-The Docker container is set up in such a way that the static-server will serve the example-client.
+*Note: Enzyme is [not ready for React 17
+yet](https://github.com/enzymejs/enzyme/issues/2429), so we're using
+an unofficial version and installing packages with `npm i
+--legacy-peer-deps` if using node v7.  We will change to an official
+version of Enzyme when a compatible released version is available.*
 
-To the example client and static server in docker run the following commands:
+## Deployment
 
-- `docker build --build-arg REACT_APP_UPLOAD_HOST="" --build-arg REACT_APP_CHAIN_ID={REACT_APP_CHAIN_ID_VALUE} --build-arg  REACT_APP_CHAIN_NAME={REACT_APP_CHAIN_NAME_VAUE}  --build-arg REACT_APP_CHAIN_HOST={REACT_APP_CHAIN_HOST} --build-arg REACT_APP_TORUS_BUILD_ENV={REACT_APP_TORUS_BUILD_ENV_VALUE} . -t example-client`
-- `docker run  --init --rm  -p 8080:8080 -v {name_for_volume}:/app/static-server/public example-client`
+The following steps should get you up and running quickly (assuming a
+working knowledge of NPM and Git):
 
-#### Docker Builds
-Docker image will be pushed from our [github repo](https://github.com/LibertyDSNP/example-client) to our [dockerhub](https://hub.docker.com/r/dsnp/example-client) when a new
-git tag is created.
+* Clone the example-client repository: `git clone git@github.com:LibertyDSNP/example-client.git`
 
-Tags will have the following format - `docker/*` i.e. `docker/v1.0.0`.
+* Install the correct npm and node version.  We recommend using asdf: `asdf install`
```

Comment:
```suggestion
* Install the correct npm and node version, which can be found in the `.tool-versions` file.  The maintainers use `asdf` for managing node and npm versions.
```
I'd rather we didn't make specific recommendations actually, but if people don't already have a version manager, it'll be easier to use asdf with this project.